src/styles/components/filter-bar.less

```diff
@@ -0,0 +1,116 @@
+.filter-bar {
+
+  display: flex;
+  flex-wrap: wrap-reverse;
+  width: 100%;
+
+  .filter-bar-left, .filter-bar-right {
+
+    display: inline-flex;
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    max-width: 100%;
+    width: auto;
+
+    .filter-bar-item {
+
+      flex: 0 0 auto;
+      max-width: 100%;
+
+    }
+
+  }
+
+  .filter-bar-right {
+
+    margin-left: auto;
+
+    &:last-child {
+
+      padding-right: 0;
+
+    }
+
+    .filter-bar-item:last-child {
+
+      // This overrides all of the specificity applied by the classes
+      // below.
+      padding-right: 0 !important;
```

Comment:
I'm sorry to bug you once again, and I'm probably missing something obvious, but I would really love to get rid of the `!important` statement.

![filter item important statement](https://cloud.githubusercontent.com/assets/647035/14748573/a2efd7cc-08ba-11e6-82d5-256641f0cdba.gif)

At least in Chrome it seems to work just fine without the `!important` statement. Could you please give it another try?